### ISSUE TITLE
I removed the nodes but the favicon remains

### ISSUE THREAD

**Issue by ztalien** [pythongosssss/ComfyUI-Custom-Scripts#311]
I don‘t like the cartoon icon so I removed this nodes, but the icon still shows in my edge tab. How can I restore it to the default? I don't like any other custom icons. Thank you!

**Comment by ztalien** [comment]
> Have you tried clearing your browser cache? Also the favicon is customisable [#169 (comment)](https://github.com/pythongosssss/ComfyUI-Custom-Scripts/issues/169#issuecomment-2016893834)

Yes, I deleted the cookies but nothing changed. Also I tried to download this nodes back again and replace the icon file, nothing happened. Maybe I made something mistake with Edge?

**Comment by s-varenia** [comment]
Where is the default favicon supposed to be?
Why did installing Node Pack change it?
That’s the only reason I removed it.
I want it back as it was
